docs/20_tutorial.md

```diff
@@ -2,3 +2,63 @@
 
 Tutorial
 ========
+
+***should the tutorial deal with the SOS establishing the risk
+limit, or any of their pre-audit responsibilities like the selection
+of which races to audit?
+
+Brief introduction to RLAs in non-technical, human friendly language.
+At the county level, something along the lines of "This tool is going
+to help you cross reference votes recorded on paper ballots with how 
+the computers read those ballots, to determine whether the computers
+did their job accurately…” Try to put their minds at ease about the
+process. :)
+
+***should the tutorial cover the mechanics of storing ballots and the
+other physical aspects of the RLA process which are not ‘in’ the RLA
+tool/system?
```

Comment:
I think not the mechanics, which vary by voting system, and even by county.
But we'd love to refer to best practices developed by counties and CDOS.